subsys/bluetooth/audio/shell/bap_broadcast_assistant.c

```diff
@@ -900,6 +902,26 @@ static int cmd_bap_broadcast_assistant_add_pa_sync(const struct shell *sh,
 		}
 	}
 
+	/* use the BASE to verify the BIS indexes set by command */
+	for (size_t j = 0U; j < CONFIG_BT_BAP_BASS_MAX_SUBGROUPS; j++) {
+		if (bis_bitfield_req == 0) {
+			/* Not set the BIS indexes by command, use BASE directly */
+			break;
+		} else if (subgroup_params[j].bis_sync == 0) {
+			/* The subgroup BIS index is not set, continue */
+			continue;
```

Comment:
Isn't this populated by the call to `bt_bap_base_foreach_subgroup`? Then it can never be 0

Reply:
is there some restrictions on broadcast source to set subgroup's bis_sync to be 0 ?

Reply:
> is there some restrictions on broadcast source to set subgroup's bis_sync to be 0 ?

Yes, each subgroup has a least one BIS, and the BIS index cannot be 0.